icu4j/perf-tests/src/main/java/com/ibm/icu/dev/test/perf/LowercaseTransliteratorPerf.java

```diff
@@ -0,0 +1,35 @@
+// © 2016 and later: Unicode, Inc. and others.
```

Comment:
This is a new file, so please update the year to 2025, and remove the `2002-2008, International Business Machines ...` block, which is legacy.